src/main/java/com/chartiq/finsemble/example/JavaExample.java

```diff
@@ -270,10 +279,14 @@ private void handleSymbol(JSONObject err, JSONObject res) {
             final String symbol = res.has("data") && res.getJSONObject("data").has("data") ?
                     res.getJSONObject("data").getString("data") :
                     "";
+            fsbl.getClients().getWindowClient().setComponentState(new JSONObject(){{put("field", "symbol");put("value", symbol);}}, this::handleSetComponentStateCb);
```

Comment:
It's easier to debug if you create objects outside of function calls:
```suggestion
            final JSONObject params = new JSONObject() {{
                put("field", "symbol");
                put("value", symbol);
            }};
            fsbl.getClients().getWindowClient().setComponentState(params, this::handleSetComponentStateCb);
```